gmonteiroc21/projeto-engenharia-de-software
# Feature/login signup

What I dit:

- Made Sign up page
- Made login page

How to test: 
- Clone de repository 
- Yarn install 
- Go to feature/login_signup branch
- Yarn install
- Yarn dev
- Verify pages (you can navegate to routes using localhost/login ou localhost/signup)
